### DISCUSSION TITLE
Getting error when running script with Appium and Serenity

### DISCUSSION THREAD

**Discussion by davidga2007** [serenity-bdd/serenity-core#2828] [Q&A]
I just started learning Appium with Serenity and trying to build the framework but stuck on this step.
 I am trying to implement General Store app into Serenity BDD but I getting the following error message when running script as Junit or as Maven:  Could not instantiate new WebDriver instance of type class org.openqa.selenium.firefox.FirefoxDriver (Cannot find firefox binary in PATH. Make sure Firefox is installed. OS appears to be: WIN10
But I even not mention Firefox anywhere.
If somebody can help me O will be appreciate.
[AppiumTest-20220523T163641Z-001.zip](https://github.com/serenity-bdd/serenity-core/files/8756494/AppiumTest-20220523T163641Z-001.zip)

**Comment by davidga2007** [comment]
@wakaleo Please any suggestions

**Reply by wakaleo** [reply]
Try asking in the Gitter channel, there are a lot of Appium users there

**Reply by davidga2007** [reply]
Thank you
